src/renderer/vt/Xterm256Engine.cpp

```diff
@@ -25,10 +25,13 @@ Xterm256Engine::Xterm256Engine(_In_ wil::unique_hfile hPipe,
 // Return Value:
 // - S_OK if we succeeded, else an appropriate HRESULT for failing to allocate or write.
 [[nodiscard]] HRESULT Xterm256Engine::UpdateDrawingBrushes(const TextAttribute& textAttributes,
-                                                           const gsl::not_null<IRenderData*> /*pData*/,
+                                                           const gsl::not_null<IRenderData*> pData,
                                                            const bool /*isSettingDefaultBrushes*/) noexcept
 {
     RETURN_IF_FAILED(VtEngine::_RgbUpdateDrawingBrushes(textAttributes));
+
+    RETURN_IF_FAILED(_UpdateHyperlinkAttr(textAttributes, pData));

```

Comment:
I don't think this will always get called at the right time unless the `HasIdenticalVisualRepresentationForBlankSpace` implementation is updated. Try making a link which is followed by spaces that are outside the linked area. I suspect those spaces will be clickable when they shouldn't be.